save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/domain/FileKey.kt

```diff
@@ -0,0 +1,58 @@
+package com.saveourtool.save.domain
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property name name of file
+ * @property uploadedMillis version of file
+ */
+@Serializable
+data class FileKey(
+    val name: String,
+    val uploadedMillis: Long,
+) {
+    /**
+     * @param fileInfo
+     */
+    constructor(fileInfo: FileInfo) : this(fileInfo.name, fileInfo.uploadedMillis)
+
+    /**
+     * @return formatted string
+     */
+    fun format(): String = "$name$FIELD_DELIMITER$uploadedMillis"
+
+    companion object {
+        const val FIELD_DELIMITER = ":"
+        const val OBJECT_DELIMITER = ","
+
+        /**
+         * @param str
+         * @return list of [FileKey]s parsed from provided string
+         */
+        fun parseList(str: String): List<FileKey> = str.split(OBJECT_DELIMITER).map { parse(it) }
```

Comment:
it's added for future: `FileKey` will be stored in `Execution.additionalFiles` instead of path to files